server/runtime_javascript_nakama.go

```diff
@@ -216,6 +216,7 @@ func (n *runtimeJavascriptNakamaModule) mappings(r *goja.Runtime) map[string]fun
 		"leaderboardRecordWrite":          n.leaderboardRecordWrite(r),
 		"leaderboardRecordDelete":         n.leaderboardRecordDelete(r),
 		"leaderboardsGetId":               n.leaderboardsGetId(r),
+		"leaderboardRecordsAroundOwner":   n.leaderboardRecordsAroundOwner(r),
```

Comment:
I think we should rename this to `tournamentRecordsHaystack` to keep it consistent with the lua runtime.
```suggestion
		"tournamentRecordsHaystack:   n.tournamentRecordsHaystack(r),
```